src/client/components/app-header/readme.md

```diff
@@ -1 +1,17 @@
 # App header
+
+- Has a space reserved for project logo
```

Comment:
I'd like these texts to be more sales-oriented. But I guess that's something the PO would have to do :)

Reply:
I think so, I'm horrible at selling things :)